server/db/populateInitial.ts

```diff
@@ -13,7 +13,7 @@ function populateUsers() {
         firstName: 'Maria', 
         lastName: 'Doe',
         username: 'mariaDoe',
-        password: bcrypt.hashSync('maria1234', salt),
```

Comment:
Both johnDoe and mariaDoe passwords were changed to have at least 8 characters

Reply:
I thought it was a mistake, I'll update postman and reset their values

Reply:
👍🏻